### PR TITLE
feat/AB#84481_allow_user_vertically_resize_code_editor_in_reference_data

### PR DESCRIPTION
**Pull request by RenzoPrats**
# Description

Feat: Allowed user to vertically resize code editor in reference data.

## Useful links

[ticket](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/84481)

## Type of change

- [X] Improvement (refactor or addition to existing functionality)

# How Has This Been Tested?

As the attached video.

## Screenshots

Peek 29-01-2024 15-40

# Checklist:

( * == Mandatory ) 

- [X] * I have set myself as assignee of the pull request
- [X] * My code follows the style guidelines of this project
- [X] * Linting does not generate new warnings
- [X] * I have performed a self-review of my own code
- [X] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [X] * I have commented my code, particularly in hard-to-understand areas
- [X] * I have put JSDoc comment in all required places
- [X] * My changes generate no new warnings
- [X] * I have included screenshots describing my changes if relevant
- [X] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by estelafs — APPROVED** [review]
Working as expected, but on small screens or when you change the width of the screen, you can see the editor is getting out of bounds of the view. Added classes to the editor to fix it.

**Comment by AntoineRelief** [comment]
@estelafs @RenzoPrats 
"vertically" 😅

I think I can easily fix that but you implemented an horizontal resize

File: apps/back-office/src/app/dashboard/pages/reference-data/reference-data.component.html [hunk 1]
```diff
@@ -53,16 +53,29 @@ <h1 class="overflow-hidden whitespace-nowrap text-ellipsis mb-0">
...
+        >
           <label class="block text-sm font-medium leading-6 text-gray-900"
             >{{ 'pages.referenceData.queryName' | translate }} *</label
           >
           <ngx-monaco-editor
+            class="!h-full !w-full"
             (onInit)="initEditor($event)"
             [options]="editorOptions"
             [formControl]="queryControl"
             class="mb-2"
```
Comment (AntoineRelief) [inline apps/back-office/src/app/dashboard/pages/reference-data/reference-data.component.html]: @RenzoPrats @estelafs 
class is defined twice so the first line is useless from an angular point of view
please really be vigilant

**Comment by AntoineRelief** [comment]
@RenzoPrats 
you really need to take care of the description of the ticket
resizing as you implemented it doesn't make much sense, as reducing the width of the component would only result in it being smaller and unreadable

**Pull request merged**